classes/Validate.php

```diff
@@ -682,7 +682,7 @@ public static function isEan13($ean13)
      */
     public static function isIsbn($isbn)
     {
-        return !$isbn || preg_match('/^[0-9-]{0,32}$/', $isbn);
+        return !$isbn || preg_match('/^[0-9][0-9-]{1,30}[0-9X]$/i', $isbn);
```

Comment:
```suggestion
        return !$isbn || preg_match(Isbn::VALID_PATTERN, $isbn);
```

And define `use PrestaShop\PrestaShop\Core\Domain\Product\ValueObject\Isbn;`